### PR TITLE
[Merged by Bors] - Initial stuff for dynamo db connector

### PR DESCRIPTION
**Pull request by simlay**
- [x] Still needs to have aws config work without `load_from_env`.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -136,7 +137,14 @@ jobs:
         rust-target:
           - x86_64-unknown-linux-musl
           - aarch64-unknown-linux-musl
-        connector-name: [test-connector, mqtt, http, postgres-source, postgres-sink, slack-sink]
+        connector-name:
```
Comment (tjtelan) [inline .github/workflows/ci.yml:140]: nice!

File: rust-connectors/sinks/dynamodb/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        let column_names: Vec<&str> = self.column_names.split(',').collect();
+        let column_types: Vec<&str> = self.column_types.split(',').collect();
+
+        if column_names.is_empty() {
+            panic!("Must have one ore more columns");
+        }
+        if column_names.len() != column_types.len() {
+            panic!("Must have the same number of column names as column types");
+        }
+        let primary_key = column_names.first().unwrap().to_string();
```
Comment (tjtelan) [inline rust-connectors/sinks/dynamodb/src/main.rs:119]: This is a good place to start, but we'll want to decouple the primary key from the order of the columns later

**Review by tjtelan — APPROVED** [review]
LGTM!

File: rust-connectors/sinks/dynamodb/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+        let dynamodb_local_config = builder.build();
+
+        let client = Client::from_conf(dynamodb_local_config);
+        let _ = self.create_table(&client).await?;
+
+        let mut stream = self.common.create_consumer_stream(0).await?;
+        info!("Starting stream");
+        while let Some(Ok(record)) = stream.next().await {
+            let _ = self.send_to_dynamodb(&record, &client).await;
```
Comment (morenol) [inline rust-connectors/sinks/dynamodb/src/main.rs:69]: I think that we should at least log if there is an error here

**Pull request closed without merging**